packages/graph-explorer/src/connector/sparql/SPARQLConnector.ts

```diff
@@ -24,13 +24,13 @@ import oneHopNeighborsBlankNodesIdsTemplate from "./templates/oneHopNeighbors/on
 import { BlankNodesMap, GraphSummary, SPARQLNeighborsRequest } from "./types";
 
 export default class SPARQLConnector extends AbstractConnector {
-  protected readonly basePath = "/sparql?query=";
+  protected readonly basePath = "/sparql";
   private readonly _summaryPath = "/rdf/statistics/summary?mode=detailed";
 
   private _blankNodes: BlankNodesMap = new Map();
 
   async fetchSchema(options?: QueryOptions): Promise<SchemaResponse> {
-    const ops = { ...options, disableCache: true };
+    const ops = { ...options, disableCache: true, method: "GET" };
```

Comment:
Is the fetchSchema request the call to the summary API? If so I suppose there are no concerns with using a GET request for this.